tools/src/main/java/org/apache/kafka/tools/streams/StreamsGroupCommand.java

```diff
@@ -154,6 +166,173 @@ private void printGroupInfo(List<GroupListing> groups) {
             }
         }
 
+        public void describeGroups() throws ExecutionException, InterruptedException {
+            String group = opts.options.valueOf(opts.groupOpt);
+            StreamsGroupDescription description = getDescribeGroup(group);
+            if (description == null)
+                return;
+            boolean verbose =  opts.options.has(opts.verboseOpt);
+            if (opts.options.has(opts.membersOpt)) {
+                printMembers(description, verbose);
+            } else if (opts.options.has(opts.stateOpt)) {
+                printStates(description, verbose);
+            } else {
+                printOffsets(description, verbose);
+            }
+        }
+
+        StreamsGroupDescription getDescribeGroup(String group) throws ExecutionException, InterruptedException {
+            DescribeStreamsGroupsResult result = adminClient.describeStreamsGroups(List.of(group));
+            Map<String, StreamsGroupDescription> descriptionMap = result.all().get();
+            return descriptionMap.get(group);
+        }
+
+        private void printMembers(StreamsGroupDescription description, boolean verbose) {
+            int groupLen = Math.max(15, description.groupId().length());
+            int maxMemberIdLen = 15, maxHostLen = 15, maxClientIdLen = 15;
+            Collection<StreamsGroupMemberDescription> members = description.members();
+            if (maybePrintEmptyGroupState(description.groupId(), description.groupState(), description.members().size())) {
+                for (StreamsGroupMemberDescription member : members) {
+                    maxMemberIdLen = Math.max(maxMemberIdLen, member.memberId().length());
+                    maxHostLen = Math.max(maxHostLen, member.processId().length());
+                    maxClientIdLen = Math.max(maxClientIdLen, member.clientId().length());
+                }
+
+                if (!verbose) {
+                    String fmt = "%" + -groupLen + "s %" + -maxMemberIdLen + "s %" + -maxHostLen + "s %" + -maxClientIdLen + "s\n";
+                    for (StreamsGroupMemberDescription member : members) {
+                        System.out.printf(fmt, "GROUP", "MEMBER", "PROCESS", "CLIENT-ID");
+                        System.out.printf(fmt, description.groupId(), member.memberId(), member.processId(), member.clientId());
+                        printTasks(member.assignment(), false);
+                        System.out.println();
+                    }
+                } else {
+                    String fmt = "%" + -groupLen + "s %s %-15s%" + -maxMemberIdLen + "s %s %15s %" + -maxHostLen + "s %" + -maxClientIdLen + "s\n";
```

Comment:
```suggestion
                    String fmt = "%" + -groupLen + "s %-23s %-15s%" + -maxMemberIdLen + "s %-15s %-15s %" + -maxHostLen + "s %" + -maxClientIdLen + "s\n";
```

Current format may get result like following:

```
GROUP           TARGET-ASSIGNMENT-EPOCH TOPOLOGY-EPOCH MEMBER          MEMBER-PROTOCOL    MEMBER-EPOCH PROCESS         CLIENT-ID      
group1          0 0              foo             streams               0 qux             bar            

```

Reply:
Wouldn't it be more readable if you used variables like `targetAssignmentEpochLength` and `topologyEpochLength`, etc. instead of numbers?

Here and in other places.